### ISSUE TITLE
[@nuxt/content-theme-docs] Global components don't work on Windows

### ISSUE THREAD

**Issue by adrianschubek** [nuxt/content#343]
### Version
@nuxt/content: 1.5.3 
@nuxt/content-theme-docs: 0.3.1
nuxt: 2.14

Node 14.7
Windows 10
### Reproduction Link
Repo created using `npx create-nuxt-app` and content module + docs theme.

https://github.com/adrianschubek/orm-ts-docs/tree/bug

#### Component in `/components/global/Test.vue`

```js
<template>
  <div>
    This is a test
  </div>
</template>

<script>
export default {

}
</script>

<style scoped>

</style>

```
#### Markdown in `/content/en/test.md`

```md
---
title: 'Introduction'
description: 'Empower your NuxtJS application with this awesome module.'
position: 1
category: 'Getting started'
version: 1.4
fullscreen: false
menuTitle: 'Intro'
---

Introducing my awesome Nuxt module!f

<alert type="info">

Check out a warning alert with a `codeblock`!

</alert>

<test></test>

```

### Steps to reproduce

1. Clone the repo
2. `npm i && npm run dev`
3. Go to `http://localhost:3000/`
4. Click Intro or visit http://localhost:3000/test --> **Error**

### What is Expected?
![image](https://user-images.githubusercontent.com/19362349/89297406-11af5b80-d664-11ea-9096-e177794b614c.png)

### What is actually happening?
Visiting http://localhost:3000/test directly or refreshing the page will result in the following.
**However** when going to http://localhost:3000/ and clicking the sidebar navigation link it _works_.

![image](https://user-images.githubusercontent.com/19362349/89297439-20960e00-d664-11ea-839e-e4eca7f71441.png)

![image](https://user-images.githubusercontent.com/19362349/89296933-62728480-d663-11ea-9ab3-f0550f88d002.png)

**Comment by adrianschubek** [comment]
I made a more minimal repo showing this issue here [adrianschubek/test-content-nuxt-issue-minimal](https://github.com/adrianschubek/test-content-nuxt-issue-minimal).
I forked the official [sandbox demo](https://codesandbox.io/s/nuxt-content-docs-theme-playground-inwxb?from-embed) and created a `components/global` folder and a `Test.vue` component. Placed the `<test></test>` tags in index.md and it results in the same issue.

**Comment by benjamincanac** [comment]
@adrianschubek I've tried your reproduction but couldn't reproduce the issue, might that be an issue with Windows? Could you try to make the reproduction on CodeSandbox directly?

**Comment by adrianschubek** [comment]
@benjamincanac  I can't reproduce this issue on [CodeSandbox](https://codesandbox.io/s/nuxt-ok-itdfq?file=/content/en/index.md) nor on a linux vm. Seems like its an issue with Windows.

However I don't have any issues when using the `nuxt/content` ([sandbox](https://codesandbox.io/s/nervous-tereshkova-tumsw?file=/nuxt.config.js)) module directly. Global components work just fine after setting `components: true` in my nuxt config.

<img src="https://user-images.githubusercontent.com/19362349/89515071-ec8f2a00-d7d6-11ea-98ee-18d2e585af8d.png" alt="alt text" width="400" height="200">

**Comment by benjamincanac** [comment]
@adrianschubek This is really weird, `@nuxt/content` registers the `components/global/` directory through [@nuxt/components hooks](https://github.com/nuxt/content/blob/dev/packages/content/lib/index.js#L62), which is the one from the theme in this case, and `@nuxt/content-theme-docs` registers the `components/global/` of the user (`rootDir`) in the [same way](https://github.com/nuxt/content/blob/dev/packages/theme-docs/src/index.js#L51).

So basically, both package add a global directory to the components.

I don't see how this could break nor how to reproduce this.

**Comment by adrianschubek** [comment]
@benjamincanac  I'm still experiencing this issue after numerous fresh installations, different node & npm versions etc.
But I'll close this issue for now as I'm not sure if its an issue with Windows in general or just my machine.

As a workaround I'll build my site using [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/about) which works great. Thanks for helping.

**Comment by adrianschubek** [comment]
Seems like several people are experiencing the same/similar issue (#363)

**Comment by chrislutw** [comment]
I have same problem, and i found the problem in \.nuxt\components\plugin.js maybe, but don't know reason.

## Version
@nuxt/content-theme-docs: 0.5.2
nuxt: 2.14.3

Node 12.16.3
Windows 7

Manual operation delete webpackChunkName windows path and it work

```js
// in \.nuxt\components\plugin.js file

IconGithubTest: () => import('../..\\components\\global\\icon\\IconGithubTest.vue' /* webpackChunkName: "D:_Projects/test-doc/components/global/icon/IconGithubTest" */).then(c => c.default || c)
// change to this
IconGithubTest: () => import('../..\\components\\global\\icon\\IconGithubTest.vue' /* webpackChunkName: "components/global/icon/IconGithubTest" */).then(c => c.default || c)
```

**Comment by f3ltron** [comment]
Same error on windows here... I am not such a fan of installing WSL2. Maybe a problem with path ?

**Comment by f3ltron** [comment]
I am using yarn workspaces and macos not working too

**Comment by Soitora** [comment]
Same here, does not work on Windows. Works on WSL2 however which is far from optimal.

**Comment by saturnonearth** [comment]
Have same issue, only thing that worked is building with WSL2 - hopefully this is fixed.

**Comment by Soitora** [comment]
It worked by modifying the lines like written here: https://github.com/nuxt/content/issues/343#issuecomment-677995382

**Comment by saturnonearth** [comment]
> It worked by modifying the lines like written here: [#343 (comment)](https://github.com/nuxt/content/issues/343#issuecomment-677995382)

This does not work for me, when replace the line, it just removes it when I run `yarn dev`

**Comment by Soitora** [comment]
> > It worked by modifying the lines like written here: [#343 (comment)](https://github.com/nuxt/content/issues/343#issuecomment-677995382)
> 
> This does not work for me, when replace the line, it just puts it back when running `yarn dev`

You need to replace when your enviroment is already live/in development (hot save), every time you modify a file it'll go back to not working.

**Comment by saturnonearth** [comment]
> > > It worked by modifying the lines like written here: [#343 (comment)](https://github.com/nuxt/content/issues/343#issuecomment-677995382)
> > 
> > 
> > This does not work for me, when replace the line, it just puts it back when running `yarn dev`
> 
> You need to replace when your enviroment is already live/in development (hot save), every time you modify a file it'll go back to not working.

Man that's pretty crazy. Ironically I tried out WSL2 and it worked but doesn't support hot-reloading 🤦‍♂️

**Comment by farnabaz** [comment]
Hey there 👋

@nuxt/content-theme-docs is moving away and will not get any updates, very soon we will open-source Docus, a brand new theme based on Nuxt 3, and Content v2 with lots of fixes, improvements, and new features.

I’m closing this issue in favor of the new docs theme. Keep an eye on https://github.com/nuxt/content/issues/726 for updates
